libs/sdk-ui-dashboard/src/model/commands/kpi.ts

```diff
@@ -73,8 +73,11 @@ export interface ChangeKpiWidgetMeasure extends IDashboardCommand {
         /**
          * Optionally specify the new header that should be used for the KPI widget with the
          * changed measure.
+         *
+         * You may specify the widget header as 'from-measure'. In that case the title will be automatically
```

Comment:
The comment mentions "from-measure" but the type has "from-title". This should be in sync, right?

Reply:
good catch; fixed, renamed to 'from-measure'